### PR TITLE
Thormods

### PR DESCRIPTION
**Pull request by thatch45**
### What does this PR do?

A few things, it fixes a bug where list configured engines would cause the master to not start, fixes the reactor starting with the master regardless of weather it is configured and adds thorium modules for runners and wheel
### Previous Behavior

Remove this section if not relevant
### New Behavior

Remove this section if not relevant
### Tests written?

No

### CODE REVIEW

File: salt/master.py [hunk 1]
```diff
@@ -542,9 +542,18 @@ def start(self):
             log.info('Creating master event publisher process')
             self.process_manager.add_process(salt.utils.event.EventPublisher, args=(self.opts,))
 
-            if 'reactor' in self.opts and 'reactor' not in self.opts['engines']:
-                log.info('Enabling the reactor engine')
-                self.opts['engines']['reactor'] = {}
+            if self.opts.get('reactor'):
```
Comment (thatch45) [inline salt/master.py]: @gtmanfred does this look ok? It is a change to your code

File: salt/master.py [hunk 2]
```diff
@@ -542,9 +542,18 @@ def start(self):
...
+                if isinstance(self.opts['engines'], list):
+                    rine = False
+                    for item in self.opts:
+                        if 'reactor' in item:
+                            rine = True
```
Comment (gtmanfred) [inline salt/master.py]: would it be good to put a break right here to to speed it up a minute?

**Comment by gtmanfred** [comment]
other than the one note, it looks good :)

This runs on the minion too yeah?

File: salt/master.py (hunk 2 above)
Comment (thatch45) [inline salt/master.py]: good catch

**Pull request merged**